boards/samr34-xpro/include/periph_conf.h

```diff
@@ -119,21 +119,21 @@ static const uart_conf_t uart_config[] = {
  */
 static const spi_conf_t spi_config[] = {
     {
-        .dev      = &(SERCOM4->SPI),
-        .miso_pin = GPIO_PIN(PC, 19),
-        .mosi_pin = GPIO_PIN(PB, 30),
-        .clk_pin  = GPIO_PIN(PC, 18),
-        .miso_mux = GPIO_MUX_F,
-        .mosi_mux = GPIO_MUX_F,
-        .clk_mux  = GPIO_MUX_F,
+        .dev      = &(SERCOM5->SPI),
+        .miso_pin = GPIO_PIN(PB, 2),
+        .mosi_pin = GPIO_PIN(PB, 22),
+        .clk_pin  = GPIO_PIN(PB, 23),
+        .miso_mux = GPIO_MUX_D,
+        .mosi_mux = GPIO_MUX_D,
+        .clk_mux  = GPIO_MUX_D,
         .miso_pad = SPI_PAD_MISO_0,
         .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
         .gclk_src = SAM0_GCLK_MAIN,
 #ifdef MODULE_PERIPH_DMA
         .tx_trigger = SERCOM4_DMAC_ID_TX,
         .rx_trigger = SERCOM4_DMAC_ID_RX,
 #endif
-    }
+    },
```

Comment:
Maybe this SPI bus can be added as second SPI bus instead, so that external SPI devices can still be connected to the previous pins? With that, `SPI_DEV(0)` needs to be replaced with `SPI_DEV(1)` in the MTD config.

I rarely work with SAM* peripheral, but I believe that the `.tx_trigger` and `.rx_trigger` have to match the SERCOM number that is used in the `.dev` member.

Reply:
Unfortunately there is only a single external SPI bus available on this board

 - https://ww1.microchip.com/downloads/en/DeviceDoc/SAM-R34-Xplained-Pro-User-Guide-DS50002803D.pdf
 - https://ww1.microchip.com/downloads/en/DeviceDoc/WLR089-Xplained-Pro-User's-Guide-DS50003040A.pdf

But good catch nonetheless - the first SPI bus is configured correctly, it's just wired internally to the LoRa radio.